assets/stylesheets/pages/_reactivex.scss

```diff
@@ -0,0 +1,11 @@
+._reactivex {
+  @extend %simple;
+
+  img {
+    max-width: 50%;
+  }
+
+  figure {
+    margin: 0;
+  }
+}
```

Comment:
On http://localhost:9292/reactivex/documentation/operators#tree, the tree isn’t formatted the same as it is [on the original site](http://reactivex.io/documentation/operators.html#tree), making it a little harder to understand and much larger vertically than the original.